packages/appsync-modelgen-plugin/src/visitors/appsync-model-introspection-visitor.ts

```diff
@@ -140,4 +141,14 @@ export class AppSyncModelIntrospectionVisitor<
     }
     throw new Error(`Unknown type ${gqlType}`);
   }
+
+  private generateModelPrimaryKeyInfo(model: CodeGenModel): PrimaryKeyInfo {
+    const primaryKeyField = this.getModelPrimaryKeyField(model);
+    const { primaryKeyType, sortKeyFields } = primaryKeyField.primaryKeyInfo!;
+    return {
+      isCustomPrimaryKey: primaryKeyType === CodeGenPrimaryKeyType.CustomId,
+      primaryKeyFieldName: this.getFieldName(primaryKeyField),
+      sortKeyFieldNames:[...sortKeyFields.map(field => this.getFieldName(field))]
```

Comment:
```suggestion
      sortKeyFieldNames: sortKeyFields.map(field => this.getFieldName(field))
```

Any particular reason to de-structure within a new array that I may be missing?